qa/zcash/full_test_suite.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python
```

Comment:
Why do we not have the assert in this file? There are a couple of other files which look like actual tests where we do not have the assert either.

Reply:
This file is used for the main test suite, whereas rpc-tests.sh runs just these rpc tests in question. I'm not inclined to tinker with this file since it is Python3 like and a bit out of scope for this ticket.

Reply:
Here is the list of the additional files which have the updated shebang, but not the assert:
`amqp_sub.py`, `test-security-check.py`, `zmq_sub.py`, `comptool.py`, `netutil.py`, `test_framework.py`, `create_wallet_200k_utxos.py`, `test-depends-sources-mirror.py`,

Reply:
Some of these do not exists int the rpc-tests directory. The scope of this ticket is specific to rpc-tests, since they are in the CI pipeline. We should make another ticket for python files outside of this dir.